chat/src/commonMain/kotlin/com/ably/chat/ChatApi.kt

```diff
@@ -98,12 +99,12 @@ internal class ChatApi(
     /**
      * Spec: CHA-M9
      */
-    suspend fun deleteMessage(roomName: String, message: Message, params: DeleteMessageParams): Message {
-        logger.trace("deleteMessage();", context = mapOf("message" to message.toString(), "params" to params.toString()))
+    suspend fun deleteMessage(roomName: String, serial: String, params: DeleteMessageParams): Message {
+        logger.trace("deleteMessage();", context = mapOf("message" to serial, "params" to params.toString()))
```

Comment:
As discussed today - should encode the `serial`

Reply:
fixed